paper/paper.tex

```diff
@@ -50,11 +50,11 @@ \subsection{The enclose API}
 \end{figure}
 
 In \emph{\RE}, the \texttt{solver} is an instance of a struct that must be a subtype of \texttt{AbstractEnclosureAlgorithm}.
-If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and implement the method \texttt{\_enclose}, which is automatically called by \texttt{enclose}.
+If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and extend the method \texttt{enclose}.
 %
 \begin{lstlisting}[language=Julia]
-_enclose(solver::MyEnclosure, f::Function,
-         D::Union{Interval, IntervalBox}; kwargs...)
+enclose(f::Function, D::Union{Interval, IntervalBox},
```

Comment:
I am not trying to make the suggestion, since I'll probably mess up indentation, but maybe we can replace with

```julia
function enclose(f::Function, dom::Union{Interval,IntervalBox}, solver::MyEnclosure; kwargs...)
   # write your algorithm implementation here
end
```

Reply:
I like the idea.